$OpenBSD: patch-configure,v 1.8 2015/05/06 08:43:04 dcoppa Exp $

/usr/lib/libreadline.so.4.0: undefined reference to `tgetnum'
/usr/lib/libreadline.so.4.0: undefined reference to `tgoto'
/usr/lib/libreadline.so.4.0: undefined reference to `tgetflag'
/usr/lib/libreadline.so.4.0: undefined reference to `tputs'
/usr/lib/libreadline.so.4.0: undefined reference to `tgetent'
/usr/lib/libreadline.so.4.0: undefined reference to `tgetstr'
collect2: error: ld returned 1 exit status

--- configure.orig	Sat May  2 21:36:29 2015
+++ configure	Wed May  6 10:33:16 2015
@@ -16762,7 +16762,7 @@ if ${ac_cv_lib_readline_rl_initialize+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lreadline  $LIBS"
+LIBS="-lreadline -lncursesw $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
